showcase/scripts/resources/BUILD.partial.bazel

```diff
@@ -0,0 +1,38 @@
+# This file is a partial recovery of the BUILD.bazel file, which was last seen on
+# https://github.com/googleapis/sdk-platform-java/commit/d59b87feddce69077c05ef8e44faea8c44f3338b
+#
+# We keep it here in order to leverage the protoc parameter inference of the hermetic build Docker image,
+# which parses BUILD.bazel files from folders with api definitions (e.g. googleapis/googleapis).
+
+java_gapic_library(
+    name = "showcase_java_gapic",
+    srcs = [":showcase_proto_with_info"],
+    gapic_yaml = None,
+    grpc_service_config = "showcase_grpc_service_config.json",
+    # TODO(#1285): Enable rest_numeric_enums once https://github.com/googleapis/gapic-showcase/issues/1255 is
+    # fixed.
+    rest_numeric_enums = False,
+    service_yaml = "showcase_v1beta1.yaml",
+    test_deps = [
```

Comment:
Can we make this file minimum? For example, I don't think we need any `test_deps` since we only parses `rest_numeric_enums`, `transport` and `include_samples`.

Reply:
Yes, I kept it as slim as possible. Thanks.